cmd/convox/apps_test.go

```diff
@@ -56,7 +104,8 @@ func TestAppsCreateWithDotsInDirName(t *testing.T) {
 
 	test.Runs(t,
 		test.ExecRun{
-			Command: "convox apps create",
+			// unset CONVOX_WAIT in case it's present in the host env
+			Command: "CONVOX_WAIT= convox apps create",
```

Comment:
We should do this globally for the whole test suite. Save the env, clear it, and restore it when done?